components/crud-web-apps/jupyter/backend/apps/common/form.py

```diff
@@ -163,7 +163,10 @@ def set_notebook_cpu(notebook, body, defaults):
     container = notebook["spec"]["template"]["spec"]["containers"][0]
 
     cpu = get_form_value(body, defaults, "cpu")
+
     cpu_limit = get_form_value(body, defaults, "cpuLimit")
+    if cpu_limit == "NaN":
```

Comment:
let's instead make a check if `NaN` is included in the value of either `cpu` or `cpu_limit`.

And if that's the case then the backend should raise a `BadRequest` werkzeug exception (400)

Reply:
@kimwnasptd Thank you for the quick review. I will merge in changes from https://github.com/kubeflow/kubeflow/pull/6058 and verify the behavior the locally and make changes as per your suggestions.